Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -384,6 +384,9 @@ - (BOOL)handleDeepLink:(NSURL *)url {
 }
 
 - (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    //always save the incoming url in the preferenceHelper, as the externalIntentURI
+    self.preferenceHelper.externalIntentURI = [userActivity.webpageURL absoluteString];
```

Comment:
I don't think we want this -- we already send this up as `universal_link_url`. Just need the URIs